DfE.FindInformationAcademiesTrusts/Extensions/StringExtensions.cs

```diff
@@ -40,4 +41,9 @@ public static string ToTitleCase(this string text)
 
         return textInfo.ToTitleCase(textInfo.ToLower(text));
     }
+
+    public static string DefaultIfNull(this string? input, string defaultValue = ViewConstants.NoDataText)
```

Comment:
```suggestion
    public static string DefaultIfNullOrWhiteSpace(this string? input, string defaultValue = ViewConstants.NoDataText)
```
Renaming this function would make it clearer that it will affect whitespace strings